### DISCUSSION TITLE
Copilot in Intellij not using whole project as reference

### DISCUSSION THREAD

**Discussion by Askath** [community/community#126242] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I am trying out the copilot chat in my Intellij IDE and noticed that @workspace does not work. I also tried similar things like /workspace etc. 

Is this still something in development? because without it chat
 feels kinda useless right now.

**Comment by davevad93** [comment]
Hi @Askath ,

I'm not using `IntelliJ`, I only use `VS Code`, `Visual Studio` and `Eclipse` so I could be wrong, but take a look at this discussion #126240 , maybe it has something to do with that. However, it's possible that the `@workspace` feature is only available for `VS Code` because it doesn't work in `Visual Studio 2022` either.

**Reply by Askath** [reply]
sadly I am still using the currently supported version not the EAP.
Is there any info on whether that will be supported soon (for other editors in general?)

**Reply by davevad93** [reply]
Hey @Askath , I haven't seen any information about it, you could take look at the docs but I doubt it says anything regarding that:

- [Using GitHub Copilot Chat in your IDE](https://docs.github.com/en/copilot/github-copilot-chat/copilot-chat-in-ides/using-github-copilot-chat-in-your-ide?tool=jetbrains)
